src/components/homePage/Testimonials.vue

```diff
@@ -1,13 +1,32 @@
 <template>
-<div id="testimonials">
-	<h2>Testimonials</h2>
-	<!-- SIDE SCROLLER -->
-</div>
+  <div id="testimonials">
+    <h2>Testimonials</h2>
+    <!-- SIDE SCROLLER -->
+    <vueper-slides>
+      <vueper-slide 
+        v-for="(slide, i) in information['slides']" 
+        :key="i" 
+        :title="slide.title" 
+        :content="slide.content" 
+        :image="slide.image" 
+      />
+    </vueper-slides>
+  </div>
 </template>
 
 <script>
+import { VueperSlides, VueperSlide } from 'vueperslides';
+import 'vueperslides/dist/vueperslides.css';
+import TestimonialInfo from "../../content/testimonials.json";
+
 export default {
 	name: 'Testimonials',
+	components: { VueperSlides, VueperSlide },
+  data () {
```

Comment:
Just curious, what does this bit of code [the `data()` function] do? @sanaasy @draymond63 

Not super familiar with Vue yet so I figured I'd ask :)

Reply:
@namansood The data section stores the state of the component. In this case the data is static (read from a JSON), but normally this is where you'd put dynamic data that can update the UI.

It's a function instead of an object purely because referencing "data" should generally return a copy of the data, not the actual data itself (stored in the same location since it's an object/non-primitive type). In cases when that's not needed, the following will work (but again, isn't really used):
`data: {
	myDynamicData: 0
}`